src/main/proto/user.proto

```diff
@@ -0,0 +1,48 @@
+syntax = "proto3";
+
+package user;
+
+import "google/protobuf/empty.proto";
+
+option java_package = "com.godeltech.grpc.user";
+option java_multiple_files = true;
+
+message UserDto {
+  uint64 id = 1;
+  string first_name = 2;
+  string last_name = 3;
+  string email = 4;
+  string phone_number = 5;
+}
+
+message UserIdDto {
+  uint64 id = 1;
+}
+
+message CreateUserRequest {
+  string first_name = 1;
+  string last_name = 2;
+  string email = 3;
+  string phone_number = 4;
+}
+
+message UpdateUserRequest {
+  UserDto user_dto = 1;
+}
+
+message FindUserByEmailRequest {
+  string email = 1;
+}
+
+message FindAllResponse {
+  repeated UserDto users = 1;
+}
+
+service UserService {
+  rpc Create(CreateUserRequest) returns (UserDto);
```

Comment:
please, follow Java naming conventions - all method names should be started with the first lowercase letter

Reply:
Reworked all message and service names